wml_test_schedule

```diff
@@ -5,7 +5,6 @@
 1 test_return_fail
 0 test_assert
 1 test_assert_fail
-1 test_assert_fail_two
```

Comment:
This test did exist, I just removed it from test_assert.cfg since it was duplicated between `test_return_fail` (tests `{RETURN ([not][true][/true][/not])}`) and `test_assert` (tests `{ASSERT ([true][/true])}`).

Reply:
Ah, ok. I suggest this addition, but will approved the PR without:
```
# Check that a typo in a test name in wml_test_schedule would be reported as a fail
9 there_is_no_test_with_this_name
```